src/utilmoneystr.h

```diff
@@ -14,6 +14,9 @@
 
 #include "amount.h"
 
+/* Do not use these functions to represent or parse monetary amounts to or from
+ * JSON but use AmountFromValue and ValueFromAmount for that.
```

Comment:
Should also fix few other things to follow this note, pls see https://github.com/UdjinM6/dash/commit/d326c8c94a8bb420e49d2dfc3c1ffb7aa39600c7.